apps/system/utils/fscmd.c

```diff
@@ -1336,6 +1337,80 @@ static int tash_df(int argc, char **args)
 }
 #endif
 
+#ifndef CONFIG_DISABLE_ENVIRON
+#if !defined(CONFIG_DISABLE_MOUNTPOINT) && \
+	!defined(CONFIG_DISABLE_PSEUDOFS_OPERATIONS) && defined(CONFIG_BCH)
+	
+enum fs_minor_num_e {
+	FS_BLOCK_MINOR_PRIMARY = 0, // minor number primary block driver, usually for /mnt
+	FS_BLOCK_MINOR_SECONDARY = 1 // usually for /ext
+};
+typedef enum fs_minor_num_e fs_minor_t;
+
+static int format_filesystem(fs_minor_t minor)
```

Comment:
Please add description for the api.

Reply:
This is internal function, not API.
So I think it doesn't need description for the API.